consensus/polybft/consensus_metrics.go

```diff
@@ -32,6 +32,9 @@ func updateBlockMetrics(currentBlock *types.Block, parentHeader *types.Header) e
 
 	// number of rounds needed to seal a block
 	metrics.SetGauge([]string{consensusMetricsPrefix, "rounds"}, float32(extra.Checkpoint.BlockRound))
+	metrics.SetGauge([]string{consensusMetricsPrefix, "block_transactions"}, float32(len(currentBlock.Transactions)))
+	metrics.SetGauge([]string{consensusMetricsPrefix, "current_block"}, float32(currentBlock.Number()))
+	metrics.IncrCounter([]string{consensusMetricsPrefix, "insert_block"}, float32(1))
```

Comment:
Is this a better name?
```suggestion
	metrics.IncrCounter([]string{consensusMetricsPrefix, "block_counter"}, float32(1))
```